src/main/C7F34.c

```diff
@@ -299,7 +379,24 @@ void func_800D91A0(struct LayerObj* arg0)
 {
 }
 
-INCLUDE_ASM("asm/us/main/nonmatchings/C7F34", func_800D91A8);
+void func_800D91A8(struct LayerObj* arg0)
+{
+    u32 var_v1;
+
+    if (g_Player.base.y_pos.i.hi < 0x201) {
+        var_v1 = (u32) ~(g_Player.base.x_pos.i.hi - 0x450) >> 0x1F;
```

Comment:
This seems like it ought to be an optimization for something but couldn't figure it out